src/api/Procfile

```diff
@@ -1,5 +1,5 @@
 web: bundle exec rails server -b 0.0.0.0
 delayed: bundle exec script/delayed_job.api.rb run
-clock: bundle exec clockworkd --log-dir=log -l -c config/clock.rb run 
-search: bundle exec rake ts:rebuild NODETACH=true
+clock: bundle exec clockworkd --log-dir=log -l -c config/clock.rb run
+search: (sleep 60 && bundle exec rake ts:rt:index) & bundle exec rake ts:rebuild NODETACH=true && fg
```

Comment:
`ts:rt:index` is not taken care of by `ts:rebuild`?

Reply:
The normal sequence for `rebuild` is:
```
task :rebuild => [
    :stop, :clear, :configure, 'ts:sql:index', :start, 'ts:rt:index'
]
```

However, running just `ts:rebuild` causes errors at this point. Only if we add `NODETACH=true` it works. And adding that makes the sequence stop in `start`, it never runs  `ts:rt:index`.

Reply:
:thought_balloon:  I really don't like this....

Reply:
I don't like it either. Any other alternative?

Reply:
In production this `ts:rt:index` job is run after the `:start` job because `NODETACH` is NOT set to `true`. In production we are running it in detached mode.

Reply:
Okay, then how about you create an `thinking_sphinx` issue and link it here?

Reply:
Done: https://github.com/pat/thinking-sphinx/issues/1150